db/migrate/20181130111723_add_volumes.rb

```diff
@@ -0,0 +1,61 @@
+class AddVolumes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :volumes, :id => :bigserial do |t|
```

Comment:
Wonder if we should use `cloud_volumes` ?  Just volumes is a little generic and with the attachments to vms I don't want to get into a situation like hardwares where its so generic it belongs to 3 different things

Reply:
hm, maybe?

Reply:
So do you want to use the cloud_ prefix for the other stuff too? Like networks, interfaces, etc.?

In theory the same volumes table can be used for infra too. Given we'd just have basic set of attributes and the rest is in extra,

Reply:
Yeah lets leave just as volume and see if when we get to infra they don't overlap at all we can split it up